.github/workflows/zap_single_visible.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     container:
       image: nycplanning/build-base:latest
     env:
-      AWS_S3_LIBRARY_BUCKET: edm-recipes
+      AWS_S3_BUCKET: edm-recipes
```

Comment:
This is something that we can drop once library is using standard dcpy s3 utils rather than its own

Reply:
since it's always the same bucket looks like it could be dropped from workflows and set in `library.__init__.py`, but happy to save that for later